charts/alfresco-audit-storage/values.yaml

```diff
@@ -0,0 +1,127 @@
+replicaCount: 1
+
+global:
+  alfrescoRegistryPullSecrets: quay-registry-secret
+
+image:
+  repository: quay.io/alfresco/alfresco-audit-storage
+  pullPolicy: IfNotPresent
+  tag: "0.0.1-A8"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+environment:
+  AUDIT_EVENTINGESTION_URI: activemq:topic:alfresco.repo.event2
+  AUDIT_EVENTINGESTION_DLQ_CONSUMEPERIOD: 60000
+  AUDIT_EVENTINGESTION_DLQ_CONSUMPTIONCOUNT: 1000
+  SPRING_PROFILES_ACTIVE: "durable-subscriptions"
+
+messageBroker:
+  # --  Broker URL formatted as per:
+  # https://activemq.apache.org/failover-transport-reference
+  url: null
+  # -- Broker username
+  username: null
+  # -- Broker password
+  password: null
+  existingConfigMap:
+    # -- Alternatively, provide message broker connection details via an existing
+    # configmap
+    name: null
+    keys:
+      # -- Key within the configmap holding the URL of the message broker
+      url: BROKER_URL
+  # -- Provide connection details alternatively via an existing secret that
+  # contains BROKER_URL, BROKER_USERNAME and BROKER_PASSWORD keys
+  existingSecret:
+    name: null
+    keys:
+      username: BROKER_USERNAME
+      password: BROKER_PASSWORD
+
+livenessProbe:
+  httpGet:
+    path: /actuator/health/liveness
+    port: http
+  initialDelaySeconds: 15
+  periodSeconds: 60
+readinessProbe:
+  httpGet:
+    path: /actuator/health/readiness
+    port: http
+  initialDelaySeconds: 15
+  periodSeconds: 60
+
+search:
```

Comment:
index?

this has really not something to do with searching capabilities

Reply:
Yeah worth renaming as it gives the impression it's tied the the full text search index as it is.